src/systemctl/systemctl.c

```diff
@@ -8472,6 +8479,13 @@ static int systemctl_parse_argv(int argc, char *argv[]) {
                         arg_reboot_argument = optarg;
                         break;
 
+                case ARG_TIMESTAMP_STYLE:
+                        if (STR_IN_SET(optarg, "us", "µs"))
+                                arg_timestamp_style = TIMESTAMP_US;
+                        if (streq(optarg, "utc"))
+                                arg_timestamp_style = TIMESTAMP_UTC;
```

Comment:
should refuse other values. and there should be "--timestamp=help" which just lists the options and exits (we usually do that nowadays for most enum-like cmdline siwtches)

Reply:
fixed both